gluetool_modules/bash_completion.py

```diff
@@ -87,7 +91,7 @@ def _add_options_from_group(options, **kwargs):
             Configurable._for_each_option(_add_option, options)
 
         # Inspect all option groups defined by the module, and add every option found
-        for module_name in self.glue.modules.iterkeys():
+        for module_name in iterkeys(self.glue.modules):
```

Comment:
Note: also this change is required https://github.com/gluetool/gluetool/pull/81/files

Reply:
I believe this one will get in "for free" - either we merge your PR first, then the change would get into this patch via rebase, or we'd merge this PR first and then your PR would get into `master` anyway.